libqtile/layout/stack.py

```diff
@@ -55,6 +55,10 @@ class Stack(Layout):
     defaults = [
         ("border_focus", "#0000ff", "Border colour(s) for the focused window."),
         ("border_normal", "#000000", "Border colour(s) for un-focused windows."),
+        ("border_focus_stack", "#881111",
+         "Border colour(s) for the focused stacked window."),
+        ("border_normal_stack", "#220000",
+         "Border colour(s) for un-focused stacked windows."),
```

Comment:
Could you make the default values `None`? That way the feature is opt-in and users won't have things suddenly change when they update.

Then below when the colors are accessed it could test for `None`s.

Reply:
I tried to do what you suggested. Please let me know if I was successful.